scripts/composer.sh

```diff
@@ -7,6 +7,9 @@ curl -sS https://getcomposer.org/installer | php
 echo "Moving composer to make it globally accessable"
 mv composer.phar /usr/bin/composer
 
+composer config -g notify-on-install false
```

Comment:
I think if we're faced with a choice of notifying or not notifying, from a package maintainer's point of view, I would prefer to be notified for all installs rather than none.

From a package consumer's point of view, as far as I can see, notifications don't contain any sensitive data, so there's no harm in notifying.